fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/literal/DateTimeV2Literal.java

```diff
@@ -247,10 +247,14 @@ public Expression plusMilliSeconds(long microSeconds) {
         return plusMicroSeconds(microSeconds * 1000L);
     }
 
-    public long getScale() {
+    public int getScale() {
         return ((DateTimeV2Type) dataType).getScale();
     }
 
+    public int commonScale(DateTimeV2Literal other) {
```

Comment:
useless function?

Reply:
Expand `TimeRoundSeries.java` and you will see